connect-widget/app/src/pages/connect/components/error-dialog.tsx

```diff
@@ -53,6 +57,11 @@ const ErrorDialog = ({ show, title, message, onClose: closed }: ErrorDialogProps
           </AlertDialogHeader>
           <AlertDialogBody>{actualMsg}</AlertDialogBody>
           <AlertDialogFooter>
+            {actualLink && (
+              <a href={actualLink} target="_blank" onClick={close}>
+                <u>See Docs</u>
```

Comment:
I looked into ModalBody, but I'm not quite sure what the use of it is. I thought since this is an AlertDialog, we shuold be using AlertDialogBody. 

Will change 'docs' to 'documentation'. The button will become quite massive though haha

I tried making it into a button of gray colour, but for some reason, it wasn't working. Chakra documentation says use ```colorScheme='gray'```, but I think it's being overwritten somewhere with ```!important```, and both buttons end up looking blue. If you insist, I can try to look into it further or maybe if you already know how to make it grey, I'll be happy to implement it!

Reply:
`ModelBody` is the ancestor of `AlertDialogBody`, if I'm not wrong. But it was just a way to represent a react node that was not too abstract and yet could hold the UI/content to be displayed. The reasoning behind it was to have the ErrorDialog to get either a string to display (simpler usage) or a piece of react code that would be rendered - this would allow us to have multiple links within the text, paragraphs, images, whatever, making the ErrorDialog more flexible. But a requirement for now, for sure.

For the button and "docs" > "documentation" we could have just "Documentation" and skip "See". But also not a big deal, we could defs go with the link if it makes it easier.

As for the color scheme/setup, I'd need to dig into it, I'm not super familiar with React to know out of my head (maybe @Orta21 knows).

Reply:
Oh, so AlertDialogBody can't take extra UI elements, but a ModalBody can. I see! That is indeed a good change to introduce then. 

I think the link looks alright - personally, I like it better than two buttons of the same color. If you don't mind, maybe we keep that for now, and if need be, we can make that change later. Changing that is more trouble than it looks :D at least for me right now!

Reply:
No worries, sounds good!

Reply:
If we're going to be using "Documentation" instead of "See docs", we should check to see that any overflow behaves as expected on a small screen width - as a sanity check

Reply:
Good point!